mason-versions.ini

```diff
@@ -8,4 +8,4 @@ clang-tidy=10.0.0
 clang-format=10.0.0
 llvm-cov=10.0.0
 binutils=2.31
-mbgl-core=1.6.0
\ No newline at end of file
+mbgl-core=1.6.0-cxx11abi
```

Comment:
Is 1.6.0-cxx11abi the final version, or some development version?

Reply:
It is indeed the final production version. It is tagged that way to be able to identify as there are no other changes apart from being rebuilt with new compiler.